### ISSUE TITLE
vscode on wayland broke again

### ISSUE THREAD

**Issue by JasonWoof** [NixOS/nixpkgs#208974]
### Describe the bug
An update in the last 10 days or so broke vscode in wayland. No error message or anything, it just doesn't run.

### Steps To Reproduce

version from 2022.11 from yesterday:
command: /nix/store/nqxc63v2jfrvxb1lybvzfy3rwhzyj56q-vscode-1.73.1/bin/code --enable-features=UseOzonePlatform --ozone-platform=wayland /home/jasonwoof/dev/wfpl2

terminal output:
Warning: 'enable-features' is not in the list of known options, but still passed to Electron/Chromium.
Warning: 'ozone-platform' is not in the list of known options, but still passed to Electron/Chromium.

result:
nothing happens. I.e. no window opens up, no running process.

version from 2022.11 from 2022-12-23
command: /nix/store/v2m6hrkvpgj19zp8wzf6s2g9h8fa79yx-vscode-1.73.1/bin/code --enable-features=UseOzonePlatform --ozone-platform=wayland /home/jasonwoof/dev/wfpl2

terminal output:
Warning: 'enable-features' is not in the list of known options, but still passed to Electron/Chromium.
Warning: 'ozone-platform' is not in the list of known options, but still passed to Electron/Chromium.

result:
vscode runs, opens a window, everything works as expected.

Please let me know if I can help. I'm an old hat at commandline and whatnot, but still sorta new to nixos, so I appreciate particularly explicit instructions when it comes to nixy things.

### Notify maintainers
@eadwu @synthetica @maxeaubrey @bobby285271

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.1, NixOS, 22.11 (Raccoon), 22.11.1347.0bf3109eeb6`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.11.1`
 - channels(jasonwoof): `"nixos-21.11pre317242.bc9b956714e, nixpkgs-21.11pre317548.e0ce3c683ae"`
 - channels(root): `"nixos-22.11, nixos-unstable"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by wegank** [comment]
Perhaps this is fixed by #208731.

**Comment by JasonWoof** [comment]
perhaps!

In any case it's working again now :)
